### ISSUE TITLE
Import fail

### ISSUE THREAD

**Issue by chouseknecht** [ansible/ansible-container#473]
##### ISSUE TYPE
 - Bug Report

##### Dockerfile
Here's the Dockerfile I'm importing

```
# Set the base image
FROM centos:latest

# File Author / Maintainer
LABEL maintainer "your name here!"
LABEL foo bar

# Add ngix and PHP repos
ADD nginx.repo /etc/yum.repos.d/nginx.repo

# Installing nginx
RUN yum -y update && \
    yum -y install epel-release \
    yum -y install nginx && \
    yum -y --enablerepo=remi,remi-php56 install nginx php-fpm php-common &&\
    yum install -y python-pip python-develop python-setuptools && \
    pip install supervisor

# Add nginx configuration files
ADD nginx.conf /etc/nginx/nginx.conf
ADD default.conf /etc/nginx/conf.d/default.conf

# Add Supervisor configuration file
ADD supervisord.conf /etc/

# Add default index.php
ADD index.php /var/www/index.php

# Expose port 80
EXPOSE 80

# Start Supervisor
CMD ["supervisord", "-n"]
```

##### SUMMARY
Importing the above returns the following `container.yml`:

```yaml
settings:
  conductor_base: centos:latest
services:
  demo:
    roles:
    - demo
```

And... the following `meta/container.yml` in the resulting role:

```yaml
# Set the base image
from: centos:latest
labels:
# File Author / Maintainer
  maintainer: '"your name here!"'
  foo: bar
# Expose port 80
ports:
- '80'
# Start Supervisor
command:
- supervisord
- -n
```

##### EXPECTED RESULTS

Maybe I'm wrong, but I expect all of the settings in `meta/container.yml` to also be in `container.yml`. Notice that `ports`, `labels`, and `command` are missing.

I am assuming that `install` still copies the contents of `meta/container.yml` from the role being installed, to the project's `container.yml`. If that's true, then I'm expecting `import` to do the same. 

In other words, the project level `container.yml` is the place of record, or source of truth, for settings.
